src/client/HostLobbyModal.ts

```
@@ -50,6 +50,40 @@ export class HostLobbyModal extends LitElement {
+
+  private handleKeyDown = (e: KeyboardEvent) => {
+    if (e.code === "Escape") {
+      e.preventDefault();
+      this.close();
+    }
+  };
+
+  connectedCallback() {
+    super.connectedCallback();
+    window.addEventListener("keydown", this.handleKeyDown);
+  }
```

Biome (2.1.2)

[error] Duplicate class member name "connectedCallback"



(lint/suspicious/noDuplicateClassMembers)